addons/crm/models/res_config_settings.py

```diff
@@ -58,6 +60,21 @@ def _inverse_pls_start_date_str(self):
             if setting.predictive_lead_scoring_start_date:
                 setting.predictive_lead_scoring_start_date_str = fields.Date.to_string(setting.predictive_lead_scoring_start_date)
 
+    @api.depends('predictive_lead_scoring_fields_str', 'predictive_lead_scoring_start_date_str')
+    def _compute_pls_help_text(self):
+        """Dynamically generates help text for better understandability, based on
+           predictive_lead_scoring_fields and predictive_lead_scoring_start_date """
+        for setting in self:
+            pls_fields_labels = ''
+            if setting.predictive_lead_scoring_fields_str:
+                pls_fields_labels = ', the ' + ', the '.join(setting.predictive_lead_scoring_fields.mapped('name'))
+            setting.predictive_lead_scoring_help_text = _('''
```

Comment:
I would not include html tags in translatable parts. Maybe build an html based on various parts to translate independently.